### ISSUE TITLE
Configurable queue name

### ISSUE THREAD

**Issue by ploeh** [GreanTech/Qaiain#14]
Right now, Qaiain looks at a queue with a hard-coded name:

``` F#
let q = storageAccount.CreateCloudQueueClient().GetQueueReference("qaiain")
```

The name should be configurable instead of being hard-coded.

**Comment by moodmosaic** [comment]
How about creating a `queue-name` entry for this in the .config file?

**Comment by ploeh** [comment]
Indeed :+1:

**Comment by ploeh** [comment]
Resolved by #15
